components/pages/demos/CarouselWrapperDemo/CarouselWrapperDemo.js

```diff
@@ -0,0 +1,342 @@
+    };
+
+    const handleOverflowFadeChange = event => {
+        setOverflowFade(event.target.checked);
+    };
+
+    const handleMaxWidthChange = event => {
+        setMaxWidth(event.target.checked);
+    };
+
+    const handleAutoTimerChange = event => {
+        setAutoTimerSeconds(+event.target.value);
+    };
+
+    const style = {
+        '--carousel-snap-position': snapPosition,
+        '--carousel-item-width': randomWidths ? 0 : 'var(--kitten-item-width)',
+    };
+
+    return (
+        <StrictMode>
+            <div
+                className={cx(styles.root, className, {
+                    [styles.maxWidth]: maxWidth,
+                    [styles.overflowFade]: overflowFade,
+                })}
+                style={style}
+            >
+                <section className={styles.section}>
+                    <div
+                        ref={gridRef}
+                        className={cx(grid.container, styles.grid)}
+                    >
+                        <Text as="h1" className={styles.headline}>
+                            Carousel Wrapper
+                        </Text>
+                        <Form
+                            align={align}
+                            activeItemIndex={activeItemIndex}
+                            infinite={infinite}
+                            randomWidths={randomWidths}
+                            maxWidth={maxWidth}
+                            autoTimerSeconds={autoTimerSeconds}
+                            onAlignChange={handleAlignChange}
+                            onActiveItemIndexChange={
+                                handleActiveItemIndexChange
+                            }
+                            onInfiniteChange={handleInfiniteChange}
+                            onRandomWidthsChange={handleRandomWidthsChange}
+                            onOverflowFadeChange={handleOverflowFadeChange}
+                            onMaxWidthChange={handleMaxWidthChange}
+                            onAutoTimerChange={handleAutoTimerChange}
+                        />
+                    </div>
+                    <div className={styles.lines} />
+                    <div
+                        ref={labelRef}
+                        data-start=""
+                        data-end=""
+                        className={styles.labels}
+                    />
+                    <div className={styles.carouselWrapper}>
+                        <CarouselWrapper
+                            ref={carouselRef}
+                            snap
+                            align={align}
+                            infinite={infinite}
+                            activeItemIndex={activeItemIndex}
+                            className={styles.carouselKittens}
+                            visibleItems={visibleItems}
+                            snapbackThreshold={isLarge ? 100 : 50}
+                            maxSnapOvershootVelocity={isLarge ? 3 : 2}
```

Comment:
If `snapbackThreshold` and `maxSnapOvershootVelocity` need to be responsive, they should be CSS vars

Reply:
it has been done, so much better!